cmd/ko/README.md

```diff
@@ -0,0 +1,146 @@
+and substituted there. For example, you might use the following in a
+Kubernetes `Deployment` resource:
+
+```yaml
+apiVersion: apps/v1beta1
+kind: Deployment
+metadata:
+  name: hello-world
+spec:
+  replicas: 1
+  template:
+    spec:
+      containers:
+      - name: hello-world
+        # This is the importpath for the Go binary to build and run.
+        image: github.com/examples/http/cmd/helloworld
+        ports:
+        - containerPort: 8080
+```
+
+**This extension of the Go convention enables us to create a tool with an
+extremely fast development cycle and effectively zero configuration.**
+
+## Usage
+
+`ko` can act as a drop-in replacement for the `go` CLI, to which it passes
+through all of the top-level `go foo` commands.
+
+`ko` also exports three commands that interact with Kubernetes resources.
+
+### `ko delete`
+
+`ko delete` simply passes through to `kubectl delete`, as with the `go`
+commands, it is exposed purely out of convenience for cleaning up resources
+created through `ko apply`.
+
+### `ko resolve`
+
+`ko resolve` builds on the [model](#the-ko-model) above to determine the set of
+Go import paths to build, containerize and publish. It then outputs a
+multi-document yaml of the resources to `STDOUT` replacing import paths with
+published image digests
+
+To determine where to publish the images, `ko` currently requires the
+environment variable `KO_DOCKER_REPO` to be set to an acceptable docker
+repository (e.g. `gcr.io/your-project`). This will likely change in a
+future version.
+
+Following the example above, this might result in:
+
+```shell
+$ KO_DOCKER_REPO=gcr.io/your-project ko resolve -f deployment.yaml
+apiVersion: apps/v1beta1
+kind: Deployment
+metadata:
+  name: hello-world
+spec:
+  replicas: 1
+  template:
+    spec:
+      containers:
+      - name: hello-world
+        # This is the digest of the published image containing the go binary
+        # at the embedded importpath.
+        image: gcr.io/your-project/github.com/examples/http/cmd/helloworld@sha256:deadbeef
+        ports:
+        - containerPort: 8080
+```
+
+*It it notable that these multi-level names are known to not be supported against
+(at least) DockerHub currently. We may employ alternate naming strategies in the
+future to broaden support, but this would sacrifice some amount of identifiability.*
+
+### `ko apply`
+
+`ko apply` is intended to parallel `kubectl apply`, but acts on the same
+resolved output as `ko resolve` emits. It is expected that `ko apply` will act
+as the vehicle for rapid iteration during development.  As changes are made to a
+particular application, users can run: `ko apply -f unit.yaml` to rapidly
+rebuild, repush and redeploy their changes.
+
+`ko apply` will invoke `kubectl apply` under the covers, and therefore apply
+to whatever `kubectl` context is active.
+
+## Relevance to Release Management
+
+`ko` is also useful for helping manage releases. For example, if your project
```

Comment:
style nit: You have a mix of style throughout this doc with "we" at first, "users" above, "your" here, and the passive voice below (e.g. release binaries may be published).

Reply:
The only instance of "We" that I could find is intended to refer to the ko maintainers.  I changed the reference below, and references to "users" to be directed at the reader ("you", "your").  LMK if I missed any.